projects/plugins/jetpack/to-test.md

```diff
@@ -1,64 +1,31 @@
-## Jetpack 10.11
+## Jetpack 11.1
 
 ### Before you start:
 
 - **At any point during your testing, remember to [check your browser's JavaScript console](https://wordpress.org/support/article/using-your-browser-to-diagnose-javascript-errors/#step-3-diagnosis) and see if there are any errors reported by Jetpack there.**
 - Use the "Debug Bar" or "Query Monitor" WordPress plugins to help make PHP notices and warnings more noticeable and report anything of note you see.
 
-### Jetpack connection sharing
+### Lazy Images [[#24643]](https://github.com/Automattic/jetpack/pull/24643)
 
-We improved how Jetpack shares the connection with Jetpack standalone plugins. To test this, try the following:
+We've updated Jetpack's [Lazy Images feature](https://jetpack.com/support/lazy-images/) to be compatible with a recent Safari browser update which enables native lazy loading of images by default. To test, try the following:
 
-- Connect Jetpack.
-- Activate a secondary plugin like Boost and/or Backup.
-- Go to Plugins.
-- Confirm the link to deactivate Jetpack says only "Deactivate" (and not 'Disconnect and Deactivate').
-- Click Deactivate.
-- Confirm the dialog shows information about the active features you have and no mention to other plugins.
-- Reconnect Jetpack and try disabling standalone plugins.
-- Make sure disconnecting from the Jetpack Dashboard still works.
+- Make sure you are using Safari version 15.4 or later to test with.
+- On a Jetpack connected site, check that Lazy Images are toggled on in: Jetpack → Settings → Performance
+- Create a test post/page with a gallery block. Add enough images for multiple rows and so that you will need to scroll down the page to view all the images (you could also use a spacer block above the gallery).
+- In Safari, view your created gallery at multiple viewport widths. Refresh the page a few times and make sure that when you are scrolling down to lazily load images, all of the images load.
+- Test with a different theme active on the site.
+- Try updating your created gallery/image block settings and make sure that all images still load as expected.
+- Give your gallery page a quick look in other browsers (e.g. Chrome, Firefox).
 
-### Lazy load
+### Contact Form [[#24450]](https://github.com/Automattic/jetpack/pull/24450/)
 
-We added Lazy Load option to the Instagram Latest Post block and improved support for Lazy Load on block based themes
+The [Contact Form](https://jetpack.com/support/jetpack-blocks/contact-form/) has undergone some maintenance cleanup. Verify expected functionality of forms remains by:
```

Comment:
Should we recommend testing with a FSE theme and the block widget area, since I think issues with the submission complete message were resolved/addressed in this update?